strings/antagonist_flavor/traitor_flavor.json

```
@@ -1,123 +1,123 @@
-        "allies": "Anyone who sees as you see, feels as you feel, may join the Champions of Evil! That means the Syndicate, the self-serving, or even the insane, as long as it has a heart of darkness, it's cool with the Champions!",
-        "goal": "You've got some napkin-note-plans for some EVIL to do today. On the side, the Champions of Evil are always looking for more morally malodorous malefactors! Get some recruiting done!",
-        "introduction": "You are the Champion of Evil.",
-        "roundend_report": "was a Champion of Evil!",
+        "allies": "Любой, кто видит то, что вы видите, чувствует то же, что вы чувствуете, может присоединиться к Чемпионам Зла! Это означает, что Синдикат, корыстный или даже безумный, пока у него есть сердце тьмы дружит с Чемпионами!",
+        "goal": "У тебя есть небольшие планы для какого-то ЗЛА, которые нужно сделать сегодня. С другой стороны, Чемпионы Зла всегда ищут более зловонных злоумышленников! Займись вербовкой!",
+        "introduction": "Ты — Чемпион Зла",
+        "roundend_report": "был Чемпионом Зла!",
         "ui_theme": "neutral",
-        "uplink": "The Champions of Evil is well connected to the black market. Your uplink has been provided for utmost evil!"
+        "uplink": "Чемпионы Зла хорошо связаны с черным рынком. Ваш канал связи предназначен для крайнего зла!"
     },
     "Corporate Climber": {
-        "allies": "Death to the Syndicate.",
-        "goal": "Killing needlessly would make you some kind of traitor, or at least definitely seen as one. This is all just a means to an end.",
-        "introduction": "You are the Corporate Climber.",
-        "roundend_report": "was a corporate climber.",
+        "allies": "Смерть Синдикату.",
+        "goal": "Убийство без необходимости сделало бы вас предателем или, по крайней мере, определенно считало бы вас таковым. Все это лишь средство для достижения цели.",
+        "introduction": "Вы — идущий по карьерной лестнице.",
+        "roundend_report": "был идущим по карьерной лестнице.",
         "ui_theme": "neutral",
-        "uplink": "You have connections to the black market for the deeds. Knock off a few loose weights, and your climb will be so much smoother."
+        "uplink": "У вас есть связи с черным рынком этих дел. Скиньте пару ненужных тяжестей, и ваш подъем станет намного более плавным."
     },
     "Cybersun Industries": {
-        "allies": "Fellow Cybersun operatives are to be trusted. Members of the MI13 organization can be trusted. All other syndicate operatives are not to be trusted.",
-        "goal": "Do not establish substantial presence on the designated facility, as larger incidents are harder to cover up.",
-        "introduction": "You are from Cybersun Industries.",
-        "roundend_report": "was a specialist from Cybersun Industries.",
+        "allies": "Сотрудникам Cybersun следует доверять. Членам организации МИ-13 можно доверять. Всем остальным оперативникам синдиката доверять нельзя.",
```

```suggestion
        "allies": "Сотрудникам Cybersun следует доверять. Членам организации MI13 можно доверять. Всем остальным оперативникам синдиката доверять нельзя.",
```
Определись, MI13 или МИ-13